integration_tests/tests/athena-query-tests.ts

```diff
@@ -16,6 +18,28 @@ describe("\nGenerate valid event and execute athena query\n", () => {
   });
 });
 
+describe("\nGenerate valid event one hour before August UTC and execute athena query\n", () => {
+  test("should contain eventId in the generated query results for August", async () => {
+    const { eventId } = await invokeFilterLambdaAndVerifyEventInS3Bucket(
+      validEventPayloadOneHourBeforeAugustUtc
+    );
+    const queryString = `SELECT * FROM "btm_transactions_standardised" where month='08'`;
+    const queryResult = await queryAthena(queryString);
+    expect(JSON.stringify(queryResult)).toContain(eventId);
+  });
+});
+
+describe("\nGenerate valid event one hour before January UTC and execute athena query\n", () => {
+  test("should not contain eventId in the generated query results for January", async () => {
+    const { eventId } = await invokeFilterLambdaAndVerifyEventInS3Bucket(
+      validEventPayloadOneHourBeforeJanuaryUtc
+    );
+    const queryString = `SELECT * FROM "btm_transactions_standardised" where month='01'`;
+    const queryResult = await queryAthena(queryString);
+    expect(JSON.stringify(queryResult)).not.toContain(eventId);
+  });
+});
```

Comment:
Considering also adding an event that's just outside the target month, and testing that it doesn't get returned.

Reply:
Sorry, that comment was meant for the preceding test.  But both tests maybe could maybe stand to have an 'inside' and 'outside' case.

Reply:
Resolved in dev standup.